pandas_datareader/wb.py

```diff
@@ -154,8 +160,18 @@ def url(self):
 
     @property
     def params(self):
-        return {'date': '{0}:{1}'.format(self.start.year, self.end.year),
-                'per_page': 25000, 'format': 'json'}
+        if self.freq == 'M':
+            return {'date': '{0}M{1:02d}:{2}M{3:02d}'.format(self.start.year,
+                    self.start.month, self.end.year, self.end.month),
+                    'per_page': 25000, 'format': 'json'}
+        if self.freq == 'Q':
+            return {'date': '{0}Q{1}:{2}Q{3}'.format(self.start.year,
+                    divmod(self.start.month-1,3)[0]+1, self.end.year,
```

Comment:
Can you change ``_sanitize_dates`` to always return ``Timestamp``, and can use ``self.start.quarter``?

- https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/base.py#L143